### DISCUSSION TITLE
Export and Import Classes

### DISCUSSION THREAD

**Discussion by ghost** [pimcore/pimcore#16750] [Q&A / Support]
Is there a convenient way to export and import classes with Pimcore? Is it enough to just copy the definition_xyz.php file and run some build command?

**Comment by dpfaffenbauer** [comment]
you can export and import them as json files in the backend or programmatically as well

**Reply by ghost** [reply]
pimcore:deployment:classes-rebuild                rebuilds db structure for classes, field collections and object bricks
    based on updated var/classes/definition_*.php files -> looks interesting too.

**Reply by dpfaffenbauer** [reply]
yes, thats the proper process for deployments. you can also run pimcore:build:classes to get the php files build without the db structure changes.
